src/tsickle.ts

```diff
@@ -476,7 +476,18 @@ abstract class ClosureRewriter extends Rewriter {
         // But it's fine to translate TS "implements Class" into Closure
         // "@extends {Class}" because this is just a type hint.
         const typeChecker = this.typeChecker;
-        const sym = this.mustGetSymbolAtLocation(impl.expression);
+        const sym = this.typeChecker.getSymbolAtLocation(impl.expression);
+        if (!sym) {
+          // It's possible for a class declaration to extend an expression that
```

Comment:
`... that does not *have* a symbol`, right?

Reply:
Done.